{
    "exceptions": {
        "client_error": {
            "message": "P\u0159i vol\u00e1n\u00ed zdroje \"{request_url}\" do\u0161lo k chyb\u011b klienta"
        },
        "decoding_error": {
            "message": "Odpov\u011b\u010f \"{request_url}\" nelze dek\u00f3dovat jako {decoding_type}"
        },
        "timeout": {
            "message": "\u010casov\u00fd limit p\u0159i vol\u00e1n\u00ed zdroje \"{request_url}\""
        }
    },
    "services": {
        "reload": {
            "description": "Znovu na\u010dte p\u0159\u00edkazy RESTful z nastaven\u00ed YAML.",
            "name": "Znovu na\u010d\u00edst"
        }
    }
}